src/components/CardItem/CardItem.test.js

```diff
@@ -10,14 +13,46 @@ function renderCardItem(args) {
     cardProp: {
       id: 0,
       text: '',
-      userName: ''
+      username: '',
+      votes: 0
```

Comment:
Votes are not passed as props so this is not really needed. Can you please remove it?